api/python/notebooks/experimental/mean_variance.ipynb

```diff
@@ -0,0 +1,726 @@
+       "    <tr>\n",
+       "      <th>1926153</th>\n",
+       "      <td>8.049836</td>\n",
+       "      <td>24762.926397</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1926155</th>\n",
+       "      <td>17.345415</td>\n",
+       "      <td>150412.440839</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2109685</th>\n",
+       "      <td>0.164319</td>\n",
+       "      <td>5.339741</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2109686</th>\n",
+       "      <td>0.368339</td>\n",
+       "      <td>24.930156</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2109687</th>\n",
+       "      <td>0.246049</td>\n",
+       "      <td>11.886186</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2109688</th>\n",
+       "      <td>0.240724</td>\n",
+       "      <td>10.307266</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2109689</th>\n",
+       "      <td>0.278420</td>\n",
+       "      <td>16.086994</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>9314 rows × 2 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                  mean       variance\n",
+       "soma_joinid                          \n",
+       "1926144      15.915025   69571.774917\n",
+       "1926146       5.972801    9471.427044\n",
+       "1926150      25.169472  139042.208628\n",
+       "1926153       8.049836   24762.926397\n",
+       "1926155      17.345415  150412.440839\n",
+       "...                ...            ...\n",
+       "2109685       0.164319       5.339741\n",
+       "2109686       0.368339      24.930156\n",
+       "2109687       0.246049      11.886186\n",
+       "2109688       0.240724      10.307266\n",
+       "2109689       0.278420      16.086994\n",
+       "\n",
+       "[9314 rows x 2 columns]"
+      ]
+     },
+     "execution_count": 22,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "experiment_name = \"mus_musculus\"\n",
+    "obs_value_filter = 'is_primary_data == True and tissue_general == \"skin of body\"'\n",
+    "\n",
+    "with cellxgene_census.open_soma(census_version=\"stable\") as census:\n",
+    "    with census[\"census_data\"][experiment_name].axis_query(\n",
+    "        measurement_name=\"RNA\", obs_query=soma.AxisQuery(value_filter=obs_value_filter)\n",
+    "    ) as query:\n",
+    "        mv_df = mean_variance(\n",
+    "            query,\n",
+    "            axis=1,\n",
+    "            calculate_mean=True,\n",
+    "            calculate_variance=True,\n",
+    "        )\n",
+    "\n",
+    "    obs_df = census[\"census_data\"][experiment_name].obs.read(coords=(query.obs_joinids(),)).concat().to_pandas()\n",
```

Comment:
since you just did a query you should just get the obs from the query, is cleaner

```
obs_df = query.obs().concat().to_pandas()
```

Reply:
Thanks, I forgot that `query` has an `obs()` method!